protostar-rust/src/lib.rs

```diff
@@ -60,7 +54,7 @@ fn find_cairo_files_in_directory(input_path: &Utf8PathBuf) -> Result<Vec<Utf8Pat
 
 fn internal_collect_tests(
     input_path: &Utf8PathBuf,
-    linked_libraries: Option<&Vec<LinkedLibrary>>,
+    linked_libraries: Option<Vec<LinkedLibrary>>,
```

Comment:
Why was this changed? We aren't consuming this value anyways, it's causing the lints to fail.

Reply:
Yes, I changed it to a ref back again, this time not nested.